service/history/historyEngine.go

```diff
@@ -357,6 +357,7 @@ func (e *historyEngineImpl) StartWorkflowExecution(startRequest *h.StartWorkflow
 		}
 	}
 	setTaskVersion(msBuilder.GetCurrentVersion(), transferTasks, timerTasks)
+	setTransferTaskTimestamp(common.NewRealTimeSource().Now(), transferTasks)
```

Comment:
Can we just have a single API for setting task information, instead of making 2 separate API calls.